### DISCUSSION TITLE
sending ibec stuck at 80.9 percent

### DISCUSSION THREAD

**Discussion by enderdrag0n422** [LukeZGD/Legacy-iOS-Kit#382] [Q&A] [answered]
i wanted to downgrade my iPhone 5s to iOS 10.3.3, but it randomly gets stuck on sending ibec 80.9%. someone please help me that would make my day.

**Comment by LukeZGD** [comment, marked as answer]
This is likely to be fixed by now since a lot of changes have been made since posting. If you haven't tried yet, maybe try again or something
